### PR TITLE
Validate election

### PR DESCRIPTION
**Pull request by cap10morgan**
Adds the Election element validations asked for in [this card](https://www.pivotaltracker.com/story/show/113421489).

### CODE REVIEW

File: test-resources/xml/v5-one-election.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0"?>
+<VipObject schemaVersion="5.0">
+  <Election id="ele001">
+  </Election>
+</VipObject>
```
Comment (tie-rack) [inline test-resources/xml/v5-one-election.xml]: There are three copies of this file. Could there just be one?

Comment (cap10morgan) [inline test-resources/xml/v5-one-election.xml]: Yep!

**Comment by cap10morgan** [comment]
@tie-rack This one look OK now?

**Comment by tie-rack** [comment]
Yeah. LGTM!

**Pull request merged**